--- a/vendor/maplibre-native/include/mbgl/storage/resource_options.hpp
+++ b/vendor/maplibre-native/include/mbgl/storage/resource_options.hpp
@@ -1,5 +1,6 @@
 #pragma once
 
+#include <cstdint>
 #include <memory>
 #include <string>
 #include <mbgl/util/tile_server_options.hpp>
--- a/vendor/maplibre-native/src/mbgl/text/bidi.hpp
+++ b/vendor/maplibre-native/src/mbgl/text/bidi.hpp
@@ -1,5 +1,6 @@
 #pragma once
 
+#include <cstdint>
 #include <set>
 #include <string>
 #include <vector>
